test/metabase/actions/test_util.clj

```diff
@@ -166,7 +166,7 @@
   "Impl for [[with-emitter]]."
   [card-or-dashboard-model {:keys [action-id], :as context} f]
   (let [parent-model (db/resolve-model card-or-dashboard-model)]
-    (mt/with-temp* [parent-model [{emitter-parent-id :id}]
+    (mt/with-temp* [parent-model [{emitter-parent-id :id} {:name (str (name parent-model) " " action-id)}]
```

Comment:
This name is used to check the emitter-usages [here](https://github.com/metabase/metabase/pull/24640/files#diff-b3ba3054398c367cc558356a9587b0dc7633e7c7f7c46a12212d04122d7b4880R36) and [here](https://github.com/metabase/metabase/pull/24640/files#diff-b3ba3054398c367cc558356a9587b0dc7633e7c7f7c46a12212d04122d7b4880R46).